crates/video/Cargo.toml

```diff
@@ -0,0 +1,41 @@
+[package]
+name = "vhost-user-video"
+version = "0.1.0"
+authors = ["Albert Esteve <aesteve@redhat.com>"]
+description = "A virtio-video device using the vhost-user protocol."
+repository = "https://github.com/rust-vmm/vhost-device"
+readme = "README.md"
+keywords = ["vhost", "video", "virt", "backend"]
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[features]
+default = ["v4l2-decoder"]
+v4l2-decoder = ["v4l2r", "nix"]
+
+[dependencies]
+bitflags = "2.3.3"
+clap = { version = "4.3",  features = ["derive"] }
+env_logger = "0.10"
+epoll = "4.3"
+enumn = "0.1.6"
+log = "0.4"
+libc = "0.2.147"
+thiserror = "1.0"
+futures = { version = "0.3", features = ["thread-pool"] }
```

Comment:
You should try and use the specific `futures_$X` crate. The whole `futures` one pulls way too many deps

Reply:
I used the same dependency line for `futures` as in `vsock` crate.

Reply:
Right, i would still replace it here with `futures_executor`. It would drop 3 deps and possibly some of their dependencies

Reply:
Great, I see, will do then :)